cns/api.go

```diff
@@ -365,3 +365,8 @@ type EndpointRequest struct {
 	HostVethName  string `json:"hostVethName"`
 	InterfaceName string `json:"InterfaceName"`
 }
+
+type GetVMUniqueIDResponse struct {
+	Response   Response `json:"response"`
```

Comment:
Do we need a `Response` here? Can the http status code instead indicate success/failure?

Reply:
I need the error message in the response but don't really care about the application return code at least for this API. I looked at couple of other API handlers in CNS and there are two patterns for returning result
1. Always return http success and then Response in the result contains the real success/failure. 
2. Return the appropriate http code but still Response is returned and validated for success case.

I am leaning towards returning correct http code + error string. So either go with #2 or go with something like this. 
type GetVMUniqueIDResponse struct {	
	VMUniqueID string   `json:"VMUniqueID"`
    Error      string   `json:"error"`
}